src/components/character-count/character-count.js

```diff
@@ -0,0 +1,195 @@
+import 'govuk-frontend/govuk/vendor/polyfills/Function/prototype/bind';
```

Comment:
Maybe worth adding a comment to link to the govuk-frontend implementation and indicate that this is intentionally a copy-and-paste.

Reply:
Done.